contracts/distribution/Distribution.sol

```diff
@@ -127,6 +129,14 @@ abstract contract Distribution is
         return true;
     }
 
+    function withdrawFees(Amount amount)
```

Comment:
why not in contract Component? if no real reason move to Component and refactor Pool, Product accordingly